### PR TITLE
Added version compatibility table

### PR DESCRIPTION
**Pull request by zsweigart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,15 @@ include ':react-native-plaid-link-sdk'
...
 ```
 
+## Version Compatibiltiy
+| React Native SDK | Android SDK | iOS SDK | |
+|---|---|---| --- |
+| 5.x.x | [2.1.0+) |   |  |
+| 4.x.x | [2.0.0-2.1.0) |   |  |
+| 3.x.x | [1.0.0-2.0.0) |   |  Deprecated |
+| 2.x.x | [0.3.0-1.0.0) |   |  Deprecated |
+| 1.x.x | [0.1.0-0.3.0) |   |  Deprecated |
```
Comment (afh) [inline README.md:87]: Add iOS SDK version compatibility for 1.x.x
```suggestion
| 1.x.x | [0.1.0-0.3.0) | <=1.1.24 |  Deprecated |
```

File: README.md [hunk 2]
```diff
@@ -77,6 +77,15 @@ include ':react-native-plaid-link-sdk'
 project(':react-native-plaid-link-sdk').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-plaid-link-sdk/android')
 ```
 
+## Version Compatibiltiy
+| React Native SDK | Android SDK | iOS SDK | |
+|---|---|---| --- |
+| 5.x.x | [2.1.0+) |   |  |
+| 4.x.x | [2.0.0-2.1.0) |   |  |
+| 3.x.x | [1.0.0-2.0.0) |   |  Deprecated |
+| 2.x.x | [0.3.0-1.0.0) |   |  Deprecated |
```
Comment (afh) [inline README.md:86]: Add iOS SDK version compatibility for 2.x.x
```suggestion
| 2.x.x | [0.3.0-1.0.0) | <=1.1.27 |  Deprecated |
```

File: README.md [hunk 3]
```diff
@@ -77,6 +77,15 @@ include ':react-native-plaid-link-sdk'
 project(':react-native-plaid-link-sdk').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-plaid-link-sdk/android')
 ```
 
+## Version Compatibiltiy
+| React Native SDK | Android SDK | iOS SDK | |
+|---|---|---| --- |
+| 5.x.x | [2.1.0+) |   |  |
+| 4.x.x | [2.0.0-2.1.0) |   |  |
+| 3.x.x | [1.0.0-2.0.0) |   |  Deprecated |
```
Comment (afh) [inline README.md:85]: Add iOS SDK version compatibility for 3.x.x
```suggestion
| 3.x.x | [1.0.0-2.0.0) | v.s. |  Deprecated |
```

File: README.md [hunk 4]
```diff
@@ -77,6 +77,15 @@ include ':react-native-plaid-link-sdk'
 project(':react-native-plaid-link-sdk').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-plaid-link-sdk/android')
 ```
 
+## Version Compatibiltiy
+| React Native SDK | Android SDK | iOS SDK | |
+|---|---|---| --- |
+| 5.x.x | [2.1.0+) |   |  |
+| 4.x.x | [2.0.0-2.1.0) |   |  |
```
Comment (afh) [inline README.md:84]: Add iOS SDK version compatibility for 4.x.x
```suggestion
| 4.x.x | [2.0.0-2.1.0) | <=1.1.33 | Active |
```

File: README.md [hunk 5]
```diff
@@ -77,6 +77,15 @@ include ':react-native-plaid-link-sdk'
 project(':react-native-plaid-link-sdk').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-plaid-link-sdk/android')
 ```
 
+## Version Compatibiltiy
+| React Native SDK | Android SDK | iOS SDK | |
+|---|---|---| --- |
+| 5.x.x | [2.1.0+) |   |  |
```
Comment (afh) [inline README.md:83]: Add iOS SDK version compatibility for 5.x.x
```suggestion
| 5.x.x | [2.1.0+) | >=1.1.34 | Active |
```

**Review by afh — CHANGES_REQUESTED** [review]
What are your thoughts on the suggestions for iOS version compatibility as well as a table header title for the last column, @zsweigart?

**Comment by zsweigart** [comment]
I just kept it simple by repeating the line

File: README.md [hunk 6]
```diff
@@ -77,6 +77,15 @@ include ':react-native-plaid-link-sdk'
 project(':react-native-plaid-link-sdk').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-plaid-link-sdk/android')
 ```
 
+## Version Compatibiltiy
+| React Native SDK | Android SDK | iOS SDK | Status |
+|---|---|---| --- |
```
Comment (afh) [inline README.md:82]: 📐 Consistent use of blankspace would be greatly appreciated by OCD people ;)
```suggestion
|---|---|---|---|
```

File: README.md [hunk 7]
```diff
@@ -77,6 +77,15 @@ include ':react-native-plaid-link-sdk'
...
+| 5.x.x | [2.1.0+) | >=1.1.34 | Active |
```
Comment (afh) [inline README.md:83]: 📐 Aligning the table columns would be greatly appreciated by OCD people ;)
```suggestion
| 5.x.x | [2.1.0+)      | >=1.1.34 | Active |
```

**Review by afh — APPROVED** [review]
Thanks for considering my suggestions, @zsweigart.
Find below some minor suggestions for consistent use of blankspace; I leave addressing these (or not) at your discretion.

**Review by afh — APPROVED** [review]
Great work @zsweigart!

**Pull request merged**